compiler/src/dotty/tools/dotc/transform/ExpandPrivate.scala

```diff
@@ -95,17 +98,4 @@ class ExpandPrivate extends MiniPhaseTransform with IdentityDenotTransformer { t
     ensurePrivateAccessible(tree.symbol)
     tree
   }
-
-  override def transformDefDef(tree: DefDef)(implicit ctx: Context, info: TransformerInfo) = {
```

Comment:
This code is still needed.
We still have methods in the same compilation unit that access private stuff.
See eg. https://github.com/lampepfl/dotty/issues/2054